parlalize/settings/defaults.py

```diff
@@ -100,13 +100,13 @@
 LAST_ACTIVITY_COUNT = 10
 
 
-#PARLALIZE votes represent
+#PARLALIZE vote options represent for vote analyses
```

Comment:
dodaj:
`# in ballots we save the option as string. this is used to transform it to a numerical value`